web_src/js/features/codeeditor.js

```diff
@@ -156,6 +167,8 @@ export async function createCodeEditor(textarea, filenameInput, previewFileModes
 
   filenameInput.addEventListener('keyup', () => {
```

Comment:
`addEventListener('input', ` instead?

Reply:
> `addEventListener('input', ` instead?

Yes 'input' is better, added a commit, thanks.